app/swagger/schemas/form526/find_rating_info_pid.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Schemas
+    module Form526
+      class FindRatingInfoPid
+        include Swagger::Blocks
+
+        swagger_schema :RatingInfo do
+          property :disability_decision_type_name, type: :string, example: 'Service Connected'
+          property :service_connected_combined_degree, type: :string, example: '90'
+          property :user_percent_of_disability, type: :string, example: '90'
```

Comment:
@omgitsbillryan I had the same conversation with one of our FE developers and thought it should be an integer as well.  I hesitated because if the user does not have a rating, "user_percent_of disability" gets passed as an empty string, instead of zero (same for "service_connected_combined_degree").  Let me know if you'd like to discuss more, as I'm not sure what I should do in this case.  Thanks!